interface/main/messages/messages.php

```diff
@@ -45,7 +45,7 @@
 require_once("$srcdir/formatting.inc.php");
 require_once("$srcdir/headers.inc.php");
 //Include Bootstrap
-call_required_libraries($bootstrap=true,$fancybox=false,$knockout=false,$datepicker=false);
+  call_required_libraries("jquery-min-3-1-1","bootstrap");
```

Comment:
<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/library/js/jquery.js"></script>

Reply:
That is 1-2-1